### ISSUE TITLE
Example should show the plot instead of showing warning

### ISSUE THREAD

**Issue by adrinjalali** [probabl-ai/skore#714]
Right now, running the example we have:

```py
$ python examples/plot_01_getting_started.py
Project file '/tmp/skore_example_jp80j2vn/my_project.skore' was successfully created.
/home/adrin/Projects/gh/me/skore/examples/plot_01_getting_started.py:105: UserWarning: FigureCanvasAgg is non-interactive, and thus cannot be shown
  plt.show()
/home/adrin/Projects/gh/me/skore/examples/plot_01_getting_started.py:158: UserWarning:

FigureCanvasAgg is non-interactive, and thus cannot be shown
```

But we should be showing the plot when run from the terminal.

**Comment by rouk1** [comment]
This may be a side effect of #647.

**Comment by augustebaum** [comment]
I managed to reproduce the error on my linux machine, but on MacOS it just works. Are you on Linux by any chance?

According to the matplotlib docs "Agg" refers to the default backend on Linux: https://matplotlib.org/stable/users/explain/figure/backends.html#selecting-a-backend
This makes me pretty sure that the issue is a quirk of matplotlib on Linux.

I tried to use PyQt6 as some StackOverflow answers suggested, but no dice. Supposedly Python comes pre-installed with Tk so the "TkAgg" backend should work, but for me the `pythom -m tkinter` test is failing.

The one success I had was to run the example with this environment variable to set the figure backend:
```sh
MPLBACKEND=webagg python plot_01_getting_started.py
```
which opens the figures in a web page.
